sources/code/renderer/preload/custombar.ts

```diff
@@ -0,0 +1,38 @@
+const { ipcRenderer } = require('electron');
+
+window.addEventListener("DOMContentLoaded", () => {
+    /*
+    * Custom title bar
+    */
+
+    const titlebar = document.createElement("div");
```

Comment:
Seriously, most errors can even be automatically fixed with `npm run lint -- --fix`.